### ISSUE TITLE
Cant connect to Ollie

### ISSUE THREAD

**Issue by baymose** [saphero/sphero-hack#124]
When trying to connect to ollie the console spits out 'beginning set up twice', but the site shows connection failed. Is there a fix for this?

**Comment by ghost** [comment]
Hi @baymose, are you able to actually connect and issue commands to your sphero/ollie despite the alert on the client-side (and are just concerned with the 'connection failed' alert displaying regardless)?

**Comment by baymose** [comment]
Nope, it never connects. The console just says that its beginning setup, and the the alert pops up.
Edit: Now I have been getting "searching..." in the console. The setup for ollie uses setup/setup-bb8.js correct?

**Comment by arialaurel** [comment]
Hmm. We never actually had an Ollie to test with when we were developing this, but according to Orbotix it should work the same as BB8. We did occasionally experience issues connecting to BB8 that were resolved by closing the app and restarting. Unfortunately we haven't had time to investigate the unreliability yet. I'm not able to buy an Ollie at the moment but when I am I'll definitely look into this. Sorry for the inconvenience!

**Comment by baymose** [comment]
No problem! 
Ill do some digging around in the code myself. I just figured maybe one of you would know. Maybe It has to be put in debug mode first? Not sure
